### ISSUE TITLE
Current quickstart documentation might be a little confusing about create_all()

### ISSUE THREAD

**Issue by cpaulik** [pallets/flask-sqlalchemy#789]
Current documentation claims that:

```
To create the initial database, just import the db object from an interactive Python shell and run the SQLAlchemy.create_all() method to create the tables and database:

>>> from yourapplication import db
>>> db.create_all()

Boom, and there is your database.
```
But the `create_all()` function does not seem to create the database, only the tables.

**Comment by RonnyPfannschmidt** [comment]
it would indeed be more clear if the word schema was  added there

**Comment by mbpreble** [comment]
Following the example in quickstart using SQLite, the database (the file `/tmp/test.db`) is created if it doesn't exist. It's not completely misleading but it's closely tied to the example use case.

It seems fair to walk back the claim since it varies by engine. If you point it at a Postgres DB which does not exist, you immediately end up with e.g.
```
psycopg2.OperationalError: FATAL:  database "database" does not exist
```

**Comment by mbpreble** [comment]
Opened a documentation PR to reduce the confusion here.

**Comment by davidism** [comment]
closed by #792
